ui/components/app/edit-gas-fee-button/index.scss

```diff
@@ -47,3 +47,10 @@
     }
   }
 }
+
+
+// Overrides react-tippy
+
+.tippy-tooltip .tippy-tooltip-content .edit-gas-fee-button__tooltip {
+  min-width: 170px;
```

Comment:
#### Without this change, the result looks like: 
<img width="385" alt="Screenshot 2022-05-17 at 6 24 27 PM" src="https://user-images.githubusercontent.com/20778143/169101966-7b7a62c2-ce5a-4724-8f05-19eafa175414.png">

#### With only a min-width of 150px, the result is:
<img width="390" alt="Screenshot 2022-05-18 at 12 03 08 PM" src="https://user-images.githubusercontent.com/20778143/169102065-008817fb-6ecd-4764-b8f1-8a4426a02500.png">